researcher_ids/RI_Claims_V1.md

```diff
@@ -60,200 +76,298 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
 interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
-Additonal terms on fields:
+### **Objects and Tokens**
+
+#### Passport
 
--   **[required]**: A field that MUST be present within a
-    [Claim Object](#claim-object-fields).
--   **[optional]** : A field that MAY be present within a
-    [Claim Object](#claim-object-fields).
--   **[optional on specific RI claims]** : A field that MAY be present only on
-    claims when where the claim definition explicitly mentions that this field
-    MAY be specified. All other standard GA4GH RI claims MUST NOT define this
-    field. This field MAY be used on
-    [custom claims](#custom-researcher-identity-claim-names) and it is
-    RECOMMENDED for the page contents within the custom claim name URL to
-    define whether this field may be present or not.
+-   A GA4GH-compatible access token, as per the [GA4GH AAI
```

Comment:
It would be elegant to focus on claim syntax and semantics in this spec and leave all protocol stuff ("access token") to the AAI spec.

Reply:
As discussed, looking to address this issue in a following PR.